README.md

```diff
@@ -183,11 +183,18 @@ Most importantly, you should see the following output from the bundle command
 when it lists the fpm gem:
 
     ...
-    Using json (1.8.1) 
+    Using json (1.8.1)
```

Comment:
trailing whitespace was stripped here, probably by some editor misconfiguration? I'll ignore it for this PR since it does'nt break anything. <3